website/source/guides/operations/plugin-backends.html.md

```diff
@@ -61,6 +61,14 @@ Put the plugin in the directory:
 $ mv my-mock-plugin /etc/vault/vault_plugins
 ```
 
+Alternatively, if you wanted a custom version of a plugin built into Vault, such as AppRole:
+
+```sh
+$ cd $GOPATH/src/github.com/hashicorp/vault/builtin/credential/approle/cmd/approle
+$ go build
```

Comment:
```suggestion
$ go build -o approle
```

Reply:
I don't think this is necessary if the cd into the approle directory.

Reply:
Yep, this was coupled to my last comment, which is now moot.